collectors/charts.d.plugin/ap/metadata.yaml

```diff
@@ -1,3 +1,5 @@
+# yamllint disable rule:line-length
```

Comment:
Guys, just update the [global yamllint config](https://github.com/netdata/netdata/blob/ea23a946ec377ec64ed4e7877d41a94f2613f1d6/.yamllint.yml#L26-L27) 😄